### ISSUE TITLE
Nginx / PHP-FPM configs aren't supervisor friendly

### ISSUE THREAD

**Issue by cdun** [kerberos-io/web#47]
Hi Folks,

I'm opening this on the web repo rather than docker because the issue is itself related to the Dockerfile contained here.

I installed Kerberos via Docker today for a quick evaluation and immediately got a whole chunk of logging from supervisord moaning that both nginx and PHP-FPM were being restarted constantly. Looking a little deeper, the Dockerfile installs both direct from APT, then registers them in supervisor.conf. Both are daemonized by default, which is something that supervisor can't manage (someone smarter may correct me but AFAI when the master process is spawned and the parent exits, the correct PID is no longer visible to supervisor).

Long story short - you probably want both to start in the foreground; the following lines inserted into the Dockerfile are a normal fix:

`RUN echo "daemon off;" >> /etc/nginx/nginx.conf`
`RUN sed -i -e "s/;daemonize\s*=\s*yes/daemonize = no/g" /etc/php5/fpm/php-fpm.conf`

Happy to send a PR if easier.

**Comment by cedricve** [comment]
Hey cdun

Thank you for your extensive comment. We hadn't the time for looking in the deamon problem, so thanks for checking the dockerfile and proposing a solution, kudos for you.

You can send a pull request to the develop branch.

Thank you for all your time cdun.

Cédric
